arch/arm/src/s32k1xx/s32k1xx_flexcan.c

```diff
@@ -979,7 +982,7 @@ static void s32k1xx_txdone(FAR void *arg)
   /* Process TX completions */
 
   uint32_t mb_bit = 1 << RXMBCOUNT;
```

Comment:
I know you didn't touch this line in this commit, but please also move it to the top.